components/server/src/user/token-service.ts

```diff
@@ -125,9 +125,10 @@ export class TokenService implements TokenProvider {
                     // Perform actual refresh
                     const stopTimer = scmTokenRefreshLatencyHistogram.startTimer({ host });
                     try {
-                        return await authProvider.refreshToken(user, requestedLifetimeDate);
-                    } finally {
+                        const result = await authProvider.refreshToken(user, requestedLifetimeDate);
```

Comment:
👋 should we add a catch, and `reportScmTokenRefreshRequest(host, opportunisticRefresh, "some_failure");`?

Reply:
I agree that feels a bit off. But error cases are handled separately below ([link](https://github.com/gitpod-io/gitpod/pull/19750/files#diff-f2ae81f929de2489e54cc19f07753a11bc982553cc55c9bb60a6f4958e956d9fR139-R154)).

Reply:
Ah, I see, an outer try/catch block - thank you!